go/Dockerfile

```diff
@@ -10,14 +10,12 @@ ADD ./go/ ./
 ENV GOCACHE=/root/.cache/go-build
 RUN --mount=type=cache,target="/root/.cache/go-build" make
 
-FROM debian:bookworm-slim as sysdb
-COPY --from=builder /build-dir/bin/coordinator .
-ENV PATH=$PATH:./
-
-CMD /bin/bash
-
-
```

Comment:
This is all unchanged, I just moved the `logservice` target to be above it in case there were any weird assumptions in this image of the `sysdb` being last.